cloudformation.go

```diff
@@ -152,29 +152,6 @@ func (c *Cloudformation) executeChangeSet(changeSetName string) error {
 	return errToReturn
 }
 
-func (c *Cloudformation) waitAndRetryIfAppropriate(startOfRetryLoop time.Time, waitFor time.Duration, recoverableErr error) (time.Duration, error) {
```

Comment:
We deleted this test here. Are we adding a new one instead? Or is this one not needed anymore?

Reply:
`waitAndRetryIfAppropriate` is no longer needed, as this is being taken care of by the `backoff` library.

The `waitForNext` method and the method testing it were used to support this. As we were able to replace the entire logic for "waiting in between retries" with the library functionality, the test is no longer needed.